README.md

```diff
@@ -1,5 +1,5 @@
-#Animate.css
-*Just-add-water CSS animation*
+#Animate.css [![GitHub release](https://img.shields.io/github/release/daneden/animate.css.svg)](https://github.com/daneden/animate.css/releases) [![Build Status](https://travis-ci.org/WarenGonzaga/animate.css.svg?branch=master)](https://travis-ci.org/WarenGonzaga/animate.css) [![devDependencies Status](https://david-dm.org/WarenGonzaga/animate.css/dev-status.svg)](https://david-dm.org/WarenGonzaga/animate.css?type=dev) [![dependencies Status](https://david-dm.org/WarenGonzaga/animate.css/status.svg)](https://david-dm.org/WarenGonzaga/animate.css) [![Built with Gulp](https://img.shields.io/badge/Built%20with-GULP-%23CF4646.svg)](http://gulpjs.com/)
+*Just-add-water CSS animation* [![GitHub forks](https://img.shields.io/github/forks/daneden/animate.css.svg?style=social&label=Fork&maxAge=2592000)](https://github.com/daneden/animate.css) [![GitHub stars](https://img.shields.io/github/stars/daneden/animate.css.svg?style=social&label=Star&maxAge=2592000)](https://github.com/daneden/animate.css) [![GitHub watchers](https://img.shields.io/github/watchers/daneden/animate.css.svg?style=social&label=Watch&maxAge=2592000)](https://github.com/daneden/animate.css) [![GitHub followers](https://img.shields.io/github/followers/daneden.svg?style=social&label=Follow&maxAge=2592000)](https://github.com/daneden)
```

Comment:
I feel like the GitHub buttons are unnecessary (assuming that most people are hitting this page on GitHub anyway!)